coil-gif/README.md

```diff
@@ -10,23 +8,38 @@ To add GIF support, import the extension library:
 implementation("io.coil-kt.coil3:coil-gif:3.0.0-rc01")
 ```
 
-And that's it! The `ImageLoader` will automatically detect any GIFs using their file headers and decode them correctly.
+And that's it! The `ImageLoader` will automatically detect any supported files using their headers and decode them correctly.
 
 Optionally, you can manually add the decoder to your component registry when constructing your `ImageLoader`:
 
 ```kotlin
+// For Android
 val imageLoader = ImageLoader.Builder(context)
     .components {
         if (SDK_INT >= 28) {
-            add(ImageDecoderDecoder.Factory())
+            add(AnimatedImageDecoder.Factory())
         } else {
             add(GifDecoder.Factory())
         }
     }
     .build()
+
+// For other platforms
+val imageLoader = ImageLoader.Builder(context)
+    .components {
+        add(AnimatedSkiaImageDecoder.Factory())
+    }
+    .build()
 ```
 
-To transform the pixel data of each frame of a GIF, see [AnimatedTransformation](/coil/api/coil-gif/coil3.gif/-animated-transformation).
+## Supported decoders and formats
+
+| Decoder                    | Supported Platforms | Supported Formats | Notes                                                                                                           |
+|----------------------------|---------------------|-------------------|-----------------------------------------------------------------------------------------------------------------|
+| `AnimatedImageDecoder`     | Android (API 28+)   | GIF, WebP, HEIF   | Powered by Android's [ImageDecoder](https://developer.android.com/reference/android/graphics/ImageDecoder) API. |
+| `GifDecoder`               | Android             | GIF               | Slower than `AnimatedImageDecoder`.                                                                             |
+| `AnimatedSkiaImageDecoder` | Other platforms     | GIF, WebP         |                                                                                                                 |
+
+## Transforms
 
-!!! Note
-    Coil includes two separate decoders to support decoding GIFs. `GifDecoder` supports all API levels, but is slower. `ImageDecoderDecoder` is powered by Android's [ImageDecoder](https://developer.android.com/reference/android/graphics/ImageDecoder) API which is only available on API 28 and above. `ImageDecoderDecoder` is faster than `GifDecoder` and supports decoding animated WebP images and animated HEIF image sequences.
+On Android, to transform the pixel data of each frame of a GIF, see [AnimatedTransformation](/coil/api/coil-gif/coil3.gif/-animated-transformation).
```

Comment:
I think we can support this on all platforms by replacing `android.graphics.Canvas` with `coil3.Canvas` in `AnimatedTransformation`.